fastlane/Fastfile

```diff
@@ -547,24 +547,53 @@ end
   #####################################################################################
   # update_certs_and_profiles
   # -----------------------------------------------------------------------------------
-  # This lane downloads all the required certs and profiles and, 
+  # This lane downloads all the required certs and profiles and,
   # if not run on CI it creates the missing ones.
   # -----------------------------------------------------------------------------------
   # Usage:
-  # bundle exec fastlane update_certs_and_profiles 
+  # bundle exec fastlane update_certs_and_profiles
   #
   # Example:
-  # bundle exec fastlane update_certs_and_profiles 
+  # bundle exec fastlane update_certs_and_profiles
   #####################################################################################
   lane :update_certs_and_profiles do | options |
     alpha_code_signing
     internal_code_signing
     appstore_code_signing
   end
 
+  lane :test_screenshots_upload do |options|
```

Comment:
This would be great to extract into a shared lane over in the release toolkit! We always upload screenshots the same way across all apps (ie – upload them, overwrite the existing ones, but DON'T PUBLISH!!! so that we can click the button by hand) 😄 

Other comments in this PR are based on the idea of moving that over.